xgi/drawing/xgi_pylab.py

```diff
@@ -152,28 +193,32 @@ def draw_xgi_nodes(ax, H, pos, node_fc, node_ec, node_lw, node_size, zorder):
         )
 
 
-def draw_xgi_hyperedges(ax, H, pos, edge_lc, edge_lw, edge_fc, d_max):
+def draw_xgi_hyperedges(ax, H, pos, edge_lc, edge_lw, edge_fc, d_max, settings):
     """Draw hyperedges.
 
     Parameters
     ----------
     ax : axis handle
-        figure axes to plot onto
+        Figure axes to plot onto
     H : Hypergraph
         A hypergraph
     pos : dict of lists
         x,y position of every node
     edge_lc : str, 4-tuple, or dict of 4-tuples or strings
-        the color of the pairwise edges
+        The color of the pairwise edges
     edge_lw : int, float, or dict
-        pairwise edge widths
+        Pairwise edge widths
     edge_fc : str, 4-tuple, ListedColormap, LinearSegmentedColormap, or dict of 4-tuples or strings
-        color of hyperedges
+        Color of hyperedges
+    settings : dict
+        Default parameters
     """
-    edge_lc = _arg_to_dict(edge_lc, H.edges)
-    edge_lw = _arg_to_dict(edge_lw, H.edges)
+    edge_lc = _color_arg_to_dict(edge_lc, H.edges, settings["edge_outline_colormap"])
```

Comment:
Maybe first check that it is not a simplicial complex
if isinstance(H, SimplicalComplex):
    raise ValueError("Use draw_xgi_complexes() instead")

Reply:
Good idea. Fixed!